elements/rhelement/gulpfile.js

```diff
@@ -12,7 +12,7 @@ const del = require("del");
 let watcher;
 
 gulp.task("clean", () => {
-  return del(["./*.compiled.*"]);
+  return del(["./*.umd.*"]);
 });
 
 gulp.task("compile", () => {
```

Comment:
@kylebuch8  Missed this during the review, but line 21 should have `compiled` changed to `umd` I think?